packages/menu-bar/src/vaadin-menu-bar-mixin.js

```diff
@@ -225,6 +225,9 @@ export const MenuBarMixin = (superClass) =>
         if (item instanceof HTMLElement && item.getAttribute('role') === 'menuitem') {
           button.appendChild(item);
           item.removeAttribute('role');
+          item.removeAttribute('aria-expanded');
+          item.removeAttribute('aria-haspopup');
+          item.removeAttribute('tabindex');
```

Comment:
Good catch, thanks 👍  

I feel like it's time to introduce a separate function like `__restoreItem(button, item)` to handle it.